atomic_reactor/plugins/exit_koji_import.py

```diff
@@ -149,6 +156,53 @@ def get_buildroot(self, worker_metadatas):
 
         return buildroots
 
+    def get_logs(self):
+        """
+        Build list of log files
+
+        :return: list, of log files
+        """
+
+        logs = None
+        output = []
+
+        # Collect logs from server
+        try:
+            logs = self.osbs.get_orchestrator_build_logs(self.build_id)
+        except OsbsException as ex:
+            self.log.error("unable to get build logs: %r", ex)
+            return output
+        except TypeError:
+            # Older osbs-client has no get_orchestrator_build_logs
+            self.log.error("OSBS client does not support get_orchestrator_build_logs")
+            return output
+
+        platforms = []
+        platform_name = {}
+        platform_logs = {}
+        for entry in logs:
+            platform = entry.platform
+            platform_logs.setdefault(platform, [])
+            if platform not in platforms:
+                platforms.append(platform)
```

Comment:
So `platform_logs` is a dict of "platform (str)" -> "file-like obj". Do you also need to keep a list of platforms? I'd expect to see something like:
```
for entry in logs:
    platform = entry.platform
    if platform not in platform_logs:
        filename = 'orchestrator' if platform is None else platform
        prefix = "%s-%s" % (self.build_id, filename)
        platform_logs[platform] = NamedTemporaryFile(prefix=prefix, ...)
    platform_logs[platform].write(entry.line)

for platform, logfile in platforms.items():
    logfile.flush()
    ...
```

Reply:
Seems to be working now, with 'if platform not in platform_logs.items():'.